src/guacamole-protocol.md

```diff
@@ -87,6 +87,12 @@ parameter.
 
 Valid protocol versions are as follows:
 
+`VERSION_1_5_0`
+: Protocol version 1.5.0 introduced two new instructions - the "msg"
+  instruction, which is used to send arbitrary messages to the client, and
+  the "name" handshake instruction, which allows the client to set the
```

Comment:
`msg` and `name` here should be within backticks so that they are represented as inline code.

Reply:
Fixed via rebase.